docker-compose.yml

```diff
@@ -168,16 +157,16 @@ services:
       - "8085:80"
     environment:
       ADMIN_PASS: "a"
-      SECRET_SALT: "h57fjemb&dn^nsJFGNjweJ"
+      SECRET_SALT: "not-secret-h57fjemb&dn^nsJFGNjweJ"
       IDP_NAME: "IDP 1"
       IDP_DOMAIN_NAME: "mfaidp"
       ID_BROKER_ACCESS_TOKEN: "dummy"
       ID_BROKER_ASSERT_VALID_IP: "false"
       ID_BROKER_BASE_URI: "dummy"
       ID_BROKER_TRUSTED_IP_RANGES: "192.168.0.1/8"
-      MFA_SETUP_URL: "http://pwmanager.local:8084/module.php/core/authenticate.php?as=ssp-hub-custom-port"
+      MFA_SETUP_URL: "http://pwmanager.local/module.php/core/authenticate.php?as=ssp-hub-custom-port"
```

Comment:
We'll probably need to find a way to re-add (or resolve the need for) these custom ports later, just as an FYI, so that it can correctly identify whether the user is headed to set up their MFA (to avoid infinitely re-prompting to force MFA set up, when used during manual testing from the host computer).